python/dgl/distributed/rpc.py

```diff
@@ -419,14 +424,14 @@ def deserialize_from_payload(cls, data, tensors):
         De-serialized object of class cls.
     """
     pos, nonarray_state = pickle.loads(data)
-    state = [None] * (len(nonarray_state) + len(tensors))
+    state = [PLACEHOLDER] * (len(nonarray_state) + len(tensors))
```

Comment:
i think you should add some comments to explain why you want to use `PLACEHOLDER`